roles/network_plugin/flannel/templates/cni-flannel.yml.j2

```diff
@@ -42,7 +42,7 @@ data:
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: kube-flannel
+  name: kube-flannel-ds-amd64
```

Comment:
To simplify multi arch support, could you use templating with a for loop in jinja ? AFAIK you just replace the arch name in 2/3 strings so It should be a good improvement...

Reply:
That only applies for those case where you have an homogeneous architecture (all nodes of the cluster only using the amd64 or arm). Unfortunately the official flannel image is offered using different tags for different architectures.

Reply:
Yes sure, I do know that but instead of copying+pasting the deamonset x time, you can use something like `{% for arch in ['amd64', 'arm64', ...] %}` or something similar to generate those...